File: Treinamento/Java/Estrutura_sequencial/Questão06.java
import java.util.Scanner;
public class Questão06 {
    public static void main(String[] args) {
        Scanner s;
        s = new Scanner(System.in);

        System.out.print("Digite o raio do círculo: ");
        int r = s.nextInt();

        System.out.printf("A área do círculo é %.3f\n", Math.PI * Math.pow(r, 2));
    }
}

File: Treinamento/C/Estrutura_sequencial/Questão06.c
#include <stdio.h>
#include <math.h>

int main() {
    const float pi = 3.1416;
    float raio;

    printf("Digite o valor do raio; ");
    scanf("%f", &raio);

    printf("Área do´círculo: %fcm²\n", pi * pow(raio, 2));    
}
